### DISCUSSION TITLE
content/daily/markdown-syntac-guide/

### DISCUSSION THREAD

**Discussion by giscus** [cha2hyun/cha2hyun.github.io#86] [Comments]
# content/posts/markdown-syntac-guide/

마크다운 신택스 가이드 by Hugo Authors

https://cha2hyun.blog/content/posts/markdown-syntac-guide/

**Comment by cha2hyun** [comment]
Giscus 테스트

**Reply by cha2hyun** [reply]
답글 테스트

**Reply by cha2hyun** [reply]
테스트 2

**Comment by cha2hyun** [comment]
Giscus Test
